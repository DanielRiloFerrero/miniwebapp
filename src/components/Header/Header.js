import React from 'react'

import { Container } from 'reactstrap'

import styles from './Header.module.scss'

const Header = () => {
  return (
    <header className={styles.header}>
      <Container>
        <a href='/'>Miniwebapp</a>
      </Container>
    </header>
  )
}

export default Header