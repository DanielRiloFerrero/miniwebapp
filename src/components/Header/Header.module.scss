@import '../../_assets/styles/abstracts/vars.scss';

.header {
  background-color: $primary-color;
  padding: $p-s 0;
}