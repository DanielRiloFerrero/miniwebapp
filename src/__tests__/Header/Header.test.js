import React from "react"
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Header from "../../components/Header/Header"

describe('Header tests', () => {
  it("render", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot()
  })
})