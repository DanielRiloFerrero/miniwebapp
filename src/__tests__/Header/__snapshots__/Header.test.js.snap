// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests render 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="container"
    >
      <a
        href="/"
      >
        Miniwebapp
      </a>
    </div>
  </header>
</div>
`;
